File: src/main/java/frc/team4069/robot/commands/OperatorControlCommandGroup.java
package frc.team4069.robot.commands;

import edu.wpi.first.wpilibj.command.CommandGroup;

// The group of commands required for operator control
public class OperatorControlCommandGroup extends CommandGroup {

    // Constructor that runs all necessary commands in parallel
    public OperatorControlCommandGroup() {
        // Add the command for driving
        addParallel(new OperatorDriveCommand());
        addParallel(new OperatorControlElevatorCommand());
    }

}

File: src/main/java/frc/team4069/robot/commands/OperatorControlCommandGroup.kt
package frc.team4069.robot.commands

import edu.wpi.first.wpilibj.command.CommandGroup

// The group of commands required for operator control
class OperatorControlCommandGroup : CommandGroup() {
    // Constructor that runs all necessary commands in parallel
    init {
        // Add the command for driving
        addParallel(OperatorDriveCommand())
        addParallel(OperatorControlElevatorCommand())
    }

}
